### PR TITLE
release: v1.0.0-beta.16

### PR DESCRIPTION
**Pull request by adamazad**
# Summary

Merges `develop` into `master`.

### CODE REVIEW

File: .env.development.local [hunk 1]
```diff
@@ -1,3 +1,4 @@
 FAST_REFRESH=true
 SKIP_PREFLIGHT_CHECK=true
+FILESYSYEM_CACHE=true
```
Comment (berteotti) [inline .env.development.local:3]: Just noticed this typo

```suggestion
FILESYSTEM_CACHE=true
```

Should I do a PR to fix this?

Comment (wixzi) [reply .env.development.local:3]: It was added to check file system cache in webpack 5. But currently that property is unused. So it wont impact this version.

**Review by wixzi — APPROVED** [review]
🍠

**Pull request merged**